gemmbench/gemm_utils.py

```diff
@@ -180,9 +195,9 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
     with tk.gen.TestLaunchContext(
         hyperparams, canonicalize=True, run=True, run_config=config
     ):
-        a = torch.randn(shape[0], shape[2], dtype=dtype)
-        b = torch.randn(shape[1], shape[2], dtype=dtype)
-        c = torch.zeros(shape[0], shape[1], dtype=torch.float32)
+        a = torch.randn(shape[0], shape[2], operand_element_type=operand_element_type)
```

Comment:
for all these torch.randn `operand_element_type` is not a valid arg to pass. It should be `dtype`. (probably just something that happened with find and replace). CI is failing because of this

Reply:
gah, I renamed too much. Thanks!